### DISCUSSION TITLE
Is there a Condition Element, child of Component Element?

### DISCUSSION THREAD

**Discussion by ArmorBearer** [wixtoolset/issues#7302] [Questions]
I had previously asked this question on the mailing list, but did not get an answer (just a snarky response):

> Since WiX v4 is not out yet, we are, of course, still using WiX v3.  I was surprised to find that the  WiX v3 documentation is relatively difficult to find on the website.  So, initially I tried looking for some information in the v4 docs.
>
> At any rate, once I found the WiX v3 documentation, I was surprised to see that the Children of the Component element includes a Condition element in v3, but not in v4 (at least not in the documentation).
>
> I find myself wondering if I should avoid modifying our v3 code to use the Condition element, since it might make the eventual transition to v4 more difficult...
>
> Can you confirm if this is an oversight in the documentation?  Or is this element being removed from the schema?

Is anyone knowledgeable about the WiX schema and able to answer this question?

This is not a question about migrating from v3 to v4.  This is a question about strategizing a solution for the future in order to meet our customer's need.

**Comment by robmen** [comment]
You are correct. Your question wasn't about migrating from v3 to v4, but the best answer is to do a migration. If something important is missing (Component Conditions are not), then you'll want to learn that sooner than later (although it is pretty late).

Note: I am one to hand out fishing poles, not fish. 😄
